tfsdk/attribute_plan_modification.go

```diff
@@ -118,6 +119,85 @@ func (r RequiresReplaceIfModifier) MarkdownDescription(ctx context.Context) stri
 	return r.markdownDescription
 }
 
+// PreserveState returns a PreserveStateModifier.
+func PreserveState() AttributePlanModifier {
+	return PreserveStateModifier{}
+}
+
+// PreserveStateModifier is an AttributePlanModifier that copies the prior state
+// value for an attribute into that attribute's plan, if that state is non-null.
+//
+// Computed attributes without the PreserveState attribute plan modifier will
+// have their value set to Unknown in the plan, so their value always will be
+// displayed as "(known after apply)" in the CLI plan output.
+// If this plan modifier is used, the prior state value will be displayed in
+// the plan instead.
```

Comment:
To match the comment from the testing.

```suggestion
// the plan instead unless a prior plan modifier adjusts the value.
```